pkg/model/constants.go

```diff
@@ -8,10 +8,11 @@ const (
 	PostgresqlPersistentVolumeName       = ApplicationName + "-postgresql-claim"
 	PostgresqlBackupPersistentVolumeName = ApplicationName + "-backup"
 	PostgresqlDeploymentName             = ApplicationName + "-postgresql"
+	KeycloakProbesName                   = ApplicationName + "-probes"
 	PostgresqlDeploymentComponent        = "database"
 	PostgresqlServiceName                = ApplicationName + "-postgresql"
 	PostgresqlImage                      = "postgres:11.5"
-	KeycloakImage                        = "quay.io/keycloak/keycloak:7.0.1"
+	KeycloakImage                        = "quay.io/keycloak/keycloak:latest"
```

Comment:
This should be tied down to a specific image or will you be doing that once a tag is cut?